packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -95,9 +97,30 @@ class WebEntrypointTarget extends Target {
     // the web_plugin_registrant.dart file alongside the generated main.dart
     const String generatedImport = 'web_plugin_registrant.dart';
 
+
+    final String? iconTreeShakerFlag = environment.defines[kIconTreeShakerFlag];
+    final AssetBundle assetBundle = AssetBundleFactory.defaultInstance(
+      logger: environment.logger,
+      fileSystem: environment.fileSystem,
+      platform: environment.platform,
+    ).createBundle();
+    final int resultCode = await assetBundle.build(
+      manifestPath: environment.projectDir.childFile('pubspec.yaml').path,
+      packagesPath: environment.projectDir.childFile('.packages').path,
+      deferredComponentsEnabled: environment.defines[kDeferredComponents] == 'true',
+      targetPlatform: TargetPlatform.web_javascript,
+    );
+    if (iconTreeShakerFlag != null) {
+      environment.defines[kIconTreeShakerFlag] = iconTreeShakerFlag;
+    }
```

Comment:
Same here

Reply:
Addressed in commit [remove outdated addition to outputs of WebReleaseBundle](https://github.com/flutter/flutter/pull/130419/commits/1fae6be4c46f79074dedab4cc261db9682f39512)